safety-architecture/tools/callgraph-tool/Makefile

```diff
@@ -58,6 +59,21 @@ install-requirements: ## Install python requirements
 	pip3 install -r requirements.txt --no-cache-dir
 	$(call target_success,$@)
 
+install-preprequisites:
+	sudo apt update
+	sudo apt install -y -q python3 \
+		build-essential cmake libstdc++-8-dev \
```

Comment:
- libstdc++-8-dev is no longer requried
- bear is only required if using it to generate compile commands database

Reply:
Removed both